src/journal-remote/journal-remote-util.c

```diff
@@ -0,0 +1,89 @@
+#include "extract-word.h"
+#include "journal-remote-util.h"
+#include "parse-util.h"
+
+void compression_args_clear(CompressionArgs *args) {
+        args->size = 0;
+        free(args->opts);
```

Comment:
`xyz_clear()` should make the input object in a _clean_ state. Here, `args->opts` is freed, but the address is still stored. Please use mfree().
```c
args->opts = mfree(args->opts);
```